### PR TITLE
Added NUnit Multiple Assembly Support.

### PR DESCRIPTION
**Pull request by RichiCoder1**
See issue #132

### CODE REVIEW

File: src/Cake.Common/Tools/NUnit/NUnitRunner.cs [hunk 1]
```diff
@@ -45,15 +46,35 @@ public void Run(FilePath assemblyPath, NUnitSettings settings)
...
+            Run(settings, GetArguments(new [] { assemblyPath }, settings), settings.ToolPath);
         }
 
-        private ProcessArgumentBuilder GetArguments(FilePath assemblyPath, NUnitSettings settings)
+        /// <summary>
+        /// Runs the tests in the specified assemblies.
+        /// </summary>
+        /// <param name="assemblyPaths">The assembly paths.</param>
+        /// <param name="settings">The settings.</param>
+        public void Run(List<FilePath> assemblyPaths, NUnitSettings settings)
```
Comment (patriksvensson) [inline src/Cake.Common/Tools/NUnit/NUnitRunner.cs]: This is very specific. Maybe `List<FilePath` could be replaced with a `IEnumerable<FilePath>`.

**Comment by RichiCoder1** [comment]
Fixed, though how in the world I squash neatly escapes me again...

**Comment by gep13** [comment]
@RichiCoder1 assuming that you have a remote setup called upstream, you will want to do the following:

```
git checkout RichiCoder1:feature/NUnit-Multiple-Assemblies
git fetch upstream
git rebase upstream/develop
```

This should get rid of your merges, and replay the actual commits that you have made onto the head of develop.  If you have any merge conflicts, you will be given the opportunity to resolve them.

You may also want to keep your fork of develop up to date as well, but you can use the same approach as above.  The technique we advocate for Chocolatey is listed here:

https://github.com/chocolatey/choco/blob/master/CONTRIBUTING.md#respond-to-feedback-on-pull-request

**Comment by patriksvensson** [comment]
@RichiCoder1 Did you get the squash to work?
Thanks @gep13 for helping out :)

**Comment by RichiCoder1** [comment]
Git makes my head hurt sometimes. Squashed.

**Comment by patriksvensson** [comment]
Perfect :+1:

**Comment by devlead** [comment]
@RichiCoder1 just fyi saw that commit 36b5fcbe379386802d174d10fdf590b3cc7a4f2b is done with an email not recognized by GitHub just go to https://github.com/settings/emails and enter any email you use and you will get recognized for all commits on your GH profile.

**Pull request merged**